### DISCUSSION TITLE
Jumper Add-on updates

### DISCUSSION THREAD

**Discussion by euu2021** [freeplane/freeplane#1012] [General]
For Jumper users that don't follow the repository: @lilive [updated Jumper](https://github.com/lilive/Freeplane-Jumper/releases).

New features are being discussed.

**Comment by lilive** [comment]
Thanks @euu2021 
Yes, after a long break I come back to work more on Jumper. Everyone is welcome to contribute to the discussions, or more.

**Comment by grotesque** [comment]
Thanks!!! I use this all the time. In my opinion jumper should be default part of freeplane.
